src/ZendServerWebApi/Model/ZendServer.php

```diff
@@ -46,8 +46,9 @@ class ZendServer
             '8.0' => '1.9',
             '8.5' => '1.10',
             '9.0' => '1.12',
-    		'9.1' => '1.14',
+	    '9.1' => '1.14',
```

Comment:
Please, fix the indentation. The rest looks good.

Reply:
Done.